proposals/stage0/REQUIREMENTS/README.md

```diff
@@ -0,0 +1,31 @@
+# Requirements
+>  Stage 0
+
+## Champion
+
+[Sendil Kumar](https://github.com/sendilkumarn)
+
+## Description
+
+This document is an outline the requirements of the Standards' representative.
+
+## Who would be responsible?
+
+The elected representative, the OpenJS Foundation
+
+## Why this proposal is important
+
+The needs of projects and ecosystem stakeholders being delegated via the representatives. These representatives are vital for voicing the collective view of and for the foundation.
+
+## Requirements of the representatives
+
+The requirements for the elected representatives are as follows:
+
+* They should be a regular member at the OpenJS Foundation.
+* They should be available to travel for the meetings.
+* They should be ready to handle the conflicts of interest that might arise.
+* They should focus / address the concerns from the OpenJS Foundation and its projects' prespective.
```

Comment:
Perhaps another duty might be to highlight agenda items of interest to project stakeholders before meetings and provide summaries of conclusions reached during the meeting.

Reply:
Good one let us add it. Thanks 👍